js/scripts.js

```diff
@@ -18,4 +18,14 @@ if ( $(window).width() <= 1024 ) {
 }
 // end main navigation
 
+  //Set nav content anchors in generated content

```

Comment:
@solenoid does this approach and function look reasonable to you?

Scenario is that the api-reference.html sub-menu `href`s for Classes, Events and Namespaces do not exist in the jsdoc html content produced by npm package `jsdoc-one-page`. i.e.
- [api-reference.html#L64](https://github.com/pubfood/pubfood.github.io/blob/develop/api-reference.html#L64)
- [api-reference.html#L70](https://github.com/pubfood/pubfood.github.io/blob/develop/api-reference.html#L70)
- [api-reference.html#L76](https://github.com/pubfood/pubfood.github.io/blob/develop/api-reference.html#L76)

Looks like the `div` references might have been added manually previously.

I use this function to so we can more easily copy 'n' paste the jsdoc generated content html from a pubfood build.

I reference `window.apiContentAnchors` defined in _api-reference.html_ so as to illustrate a little closer to the source use of the script.